controllers/error-controller.js

```diff
@@ -0,0 +1,17 @@
+const handle400s = (error, request, response, next) => {
+  if (error.code === "23502") {
+    response.status(400).send({ msg: "Bad Request" });
+  } else {
+    next(error);
+  }
+};
+
+const handleCustomErrors = (error, request, response, next) => {
+  if (error.status && error.msg) {
+    response.status(error.status).send({ msg: error.msg });
+  } else {
+    next(error);
+  }
+};
```

Comment:
We don't need these error handlers yet (see my comment in your model), so these can be removed for the time being.